lang/solidity/Portfile

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           boost 1.0
+
+github.setup        ethereum solidity 0.8.19 v
+github.tarball_from downloads
```

Comment:
You are not downloading from "downloads", an [ancient GitHub mechanism discontinued a decade ago](https://github.blog/2012-12-12-goodbye-uploads/). You are downloading from "releases".

```suggestion
```